CONTRIBUTING.md

```diff
@@ -79,17 +79,17 @@ If everything looks good, they will merge the code and release a new version whi
 
 <!-- markdown-link-check-disable -->
 
-[pull requests]: https://github.com/mineiros-io/terraform-google-bigquery-dataset/pulls
-[pre-commit-file]: https://github.com/mineiros-io/terraform-google-bigquery-dataset/blob/main/.pre-commit-config.yaml
-
-<!-- markdown-link-check-enable -->
-
 [github flow]: https://guides.github.com/introduction/flow/
 [codeowners]: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 [fork]: https://help.github.com/en/github/getting-started-with-github/fork-a-repo
-[create a new branch]: https://guides.github.com/introduction/flow/
-[readme driven development]: https://tom.preston-werner.com/2010/08/23/readme-driven-development.html
 [commit]: https://help.github.com/en/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project
+[create a new branch]: https://guides.github.com/introduction/flow/
 [create a pull request]: https://help.github.com/articles/creating-a-pull-request/
+
```

Comment:
this isn't the latest version of `CONTRIBUTING.md`, please see https://github.com/mineiros-io/terraform-module-template/pull/164 for details